.github/workflows/labbox-ephys-install-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: labbox-ephys-install-tests
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - master
+defaults:
+  run:
+    shell: bash -l {0}
```

Comment:
Important to use a login shell so that the conda environment gets activated. See: https://github.com/conda-incubator/setup-miniconda/issues/128